ocs_ci/utility/environment_check.py

```diff
@@ -85,73 +118,58 @@ def get_status_after_execution():
     """
     get_environment_status(ENV_STATUS_POST)
 
-    pod_diff = DeepDiff(
+    pod_diff = compare_dicts(
         ENV_STATUS_PRE['pod'], ENV_STATUS_POST['pod']
     )
-    sc_diff = DeepDiff(
+    sc_diff = compare_dicts(
         ENV_STATUS_PRE['sc'], ENV_STATUS_POST['sc']
     )
-    cephfs_diff = DeepDiff(
+    cephfs_diff = compare_dicts(
         ENV_STATUS_PRE['cephfs'], ENV_STATUS_POST['cephfs']
     )
-    cephbp_diff = DeepDiff(
+    cephbp_diff = compare_dicts(
         ENV_STATUS_PRE['cephbp'], ENV_STATUS_POST['cephbp']
     )
-    pv_diff = DeepDiff(
+    pv_diff = compare_dicts(
         ENV_STATUS_PRE['pv'], ENV_STATUS_POST['pv']
     )
-    pvc_diff = DeepDiff(
+    pvc_diff = compare_dicts(
         ENV_STATUS_PRE['pvc'], ENV_STATUS_POST['pvc']
     )
-    secret_diff = DeepDiff(
+    secret_diff = compare_dicts(
         ENV_STATUS_PRE['secret'], ENV_STATUS_POST['secret']
     )
-    namespace_diff = DeepDiff(
+    namespace_diff = compare_dicts(
         ENV_STATUS_PRE['namespace'], ENV_STATUS_POST['namespace']
     )
     diffs_dict = {
         'pods': pod_diff,
-        'sc': sc_diff,
+        'storageClasses': sc_diff,
         'cephfs': cephfs_diff,
         'cephbp': cephbp_diff,
         'pvs': pv_diff,
         'pvcs': pvc_diff,
-        'secret': secret_diff,
-        'ns': namespace_diff,
+        'secrets': secret_diff,
+        'namespaces': namespace_diff,
     }
     leftover_detected = False
 
     leftovers = {'Leftovers added': [], 'Leftovers removed': []}
     for kind, kind_diff in diffs_dict.items():
-        if ADDED_RESOURCE in kind_diff:
-            try:
-                leftovers['Leftovers added'].append({
-                    kind: kind_diff[ADDED_RESOURCE][
-                        ''.join(kind_diff[ADDED_RESOURCE])
-                    ]
-                })
-            except KeyError:
-                leftovers['Leftovers added'].append({
-                    kind: kind_diff[ADDED_RESOURCE]
-                })
+        if kind_diff[0]:
+            leftovers[
+                'Leftovers added'
+            ].append({f"***{kind}***": kind_diff[0]})
```

Comment:
I would drop `*` symbols so it wouldn't look like this:
```
E           Resources added:
E           - '***pods***':
E             - apiVersion: v1
E               kind: Pod
```
But more like:
```
E           Resources added:
E           - pods:
E             - apiVersion: v1
E               kind: Pod
```
It feels more consistent with `yaml` format for me but that's just my opinion.

Reply:
I got feedback from ppl that when the output is long, they can't seem to understand what the resource leftovers it